src/sections/brand-header.html

```diff
@@ -2,6 +2,4 @@
   <div class="brand-header-wrapper">
     <div class="brand-header ${splitView ? 'split-view' : ''}">
       <img class="logo" src="res/openapi-designer-logo.svg"/>
-    </div>
```

Comment:
Why the closing div -tags are removed from here?

Reply:
Propably removed them accidentally and didn't notice it, my bad!